delayed_spans.go

```diff
@@ -0,0 +1,77 @@
+// (c) Copyright IBM Corp. 2022
+
+package instana
+
+import (
+	"net/url"
+)
+
+const maxDelayedSpans = 500
+
+var delayed = &delayedSpans{
+	spans: make(chan *spanS, maxDelayedSpans),
+}
+
+type delayedSpans struct {
+	spans chan *spanS
+}
+
+func (ds *delayedSpans) append(span *spanS) bool {
+	select {
+	case ds.spans <- span:
+		return true
+	default:
+		return false
+	}
+}
+
```

Comment:
Suggestion: Addition of short function description as comment would have been nice

Reply:
Done